doc/release/platforms/README.ibm.rst

```diff
@@ -96,9 +103,9 @@ the following settings:
    site to another so check with your site's documentation for
    details.
 
-   We started work on a loadleveler launcher (see README.launcher for
+   We started work on a loadleveler launcher (see :ref:`readme-launcher` for
    a general description of the role of launchers in Chapel) which can
-   be utilized by setting the ``CHPL_LAUNCH`` environment variable to
+   be utilized by setting the ``CHPL_LAUNCHER`` environment variable to
    ``loadleveler``.  At the time of the release, this launcher was not
```

Comment:
The paragraph itself sounds too much like "we just did this" and that contradicts the top level warning, or at least would lead to confusion.  Perhaps if parts of this paragraph were reworded to "in the past" or something . . .